File: beecrowd/.vscode/1555.py
n=int(input())
while n!=0:
    x,y=input().split()
    x=int(x)
    y=int(y)
    r=9*x*x + y*y
    b=2*x*x + 25*y*y
    c=-100*x+y*y*y
    if r>b and r>c:
        print("Rafael ganhou")
    elif b>r and b>c:
        print("Beto ganhou")
    else:
        print("Carlos ganhou")
    n-=1

File: beecrowd/1555.cpp
#include<iostream>
using namespace std;
int main()
{
    int n;
    cin>>n;
    while (n--)
    {
        int x,y;
        cin>>x>>y;
        int r=9*x*x + y*y;
        int b=2*x*x + 25*y*y;
        int c=-100*x+y*y*y;
        if(r>b && r>c)
        {
            cout<<"Rafael ganhou"<<endl;
        }
        else if(b>r && b>c)
        {
            cout<<"Beto ganhou"<<endl;
        }
        else 
        {
            cout<<"Carlos ganhou"<<endl;
        }

    }
    
}